var/spack/repos/builtin/packages/gsl-lite/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class GslLite(CMakePackage):
+    """A single-file header-only version of ISO C++ Guidelines Support Library
+    (GSL) for C++98, C++11, and later"""
+
+    homepage = "https://github.com/gsl-lite/gsl-lite"
+    git = "https://github.com/gsl-lite/gsl-lite.git"
+    url = "https://github.com/gsl-lite/gsl-lite/archive/refs/tags/v0.38.1.tar.gz"
+
+    maintainers = ["kgerheiser", "edwardhartnett", "Hang-Lei-NOAA"]
```

Comment:
There is new syntax for this (see https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers).

Reply:
I'll use that and also update the list to what it should be today, thanks!